### ISSUE TITLE
When training the Fractal dataset, the loss curve is abnormal.

### ISSUE THREAD

**Issue by WJH404-JNU** [shihao1895/MemoryVLA#22]
When training the Fractal dataset, the loss was very low at the beginning (0.0362), and then it remained within a certain range (0.1 to 0.6) throughout the process. Could you please show me the loss curve during your training of the Fractal dataset?

**Comment by shihao1895** [comment]
Yes, this is a normal phenomenon.

The diffusion action expert loss is not a reliable indicator of manipulation success rate. It typically drops quickly to a relatively small value and then fluctuates within a certain range.

The loss is already low at the beginning because the Open-X embodiment pretraining data includes Fractal, so the model is partially pretrained on it.

My training loss curve on Fractal is shown below:

<img width="1200" height="700" alt="Image" src="https://github.com/user-attachments/assets/0307c35e-4663-41fd-b20b-f36f3ccee623" />
